src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -159,6 +159,293 @@
+    type: string
+    short-summary: Action to perform on headers. Possible values are Append, Overwrite, Delete.
+  - name: --redirect-type
+    type: string
+    short-summary: The redirect type the rule will use when redirecting traffic. Possible values are Moved, Found, TemporaryRedirect and PermanentRedirect.
+  - name: --redirect-protocol
+    type: string
+    short-summary: Protocol to use for the redirect. Possible values are MatchRequest, Http, Https.
+  - name: --custom-hostname
+    type: string
+    short-summary: Host to redirect. Leave empty to use the incoming host as the destination host.
+  - name: --custom-path
+    type: string
+    short-summary: The full path to redirect. Path cannot be empty and must start with /. Leave empty to use the incoming path as destination path.
+  - name: --custom-querystring
+    type: string
+    short-summary: The set of query strings to be placed in the redirect URL. leave empty to preserve the incoming query string.
+  - name: --custom-fragment
+    type: string
+    short-summary: Fragment to add to the redirect URL.
+  - name: --query-string-behavior
+    type: string
+    short-summary: Caching behavior for the requests. Possible values are Include, IncludeAll, Exclude and ExcludeAll.
+  - name: --query-parameters
+    type: string
+    short-summary: query parameters to include or exclude (comma separated).
+  - name: --source-pattern
+    type: string
+    short-summary: define a request URI pattern that identifies the type of requests that may be rewritten.
+  - name: --destination
+    type: string
+    short-summary: Define the destination path for be used in the rewrite. This will overwrite the source pattern.
+  - name: --preserve-unmatched-path
+    type: boolean
+    short-summary: If True, the remaining path after the source pattern will be appended to the new destination path.
+examples:
+  - name: Create a global rule to disable caching.
+    text: >
+        az cdn endpoint rule add -g group -n endpoint --profile-name profile --order 0\\
+            --rule-name global --action-name CacheExpiration --cache-behavior BypassCache
+  - name: Create a rule for http to https redirect
+    text: >
+        az cdn endpoint rule add -g group -n endpoint --profile-name profile --order 1\\
+            --rule-name "redirect" --match-variable RequestScheme --operator Equal --match-values HTTPS\\
+            --action-name "UrlRedirect" --redirect-protocol Https --redirect-type Moved
+"""
+
+helps['cdn endpoint rule remove'] = """
+type: command
+short-summary: Remove a delivery rule from an endpoint.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+examples:
+  - name: Remove the global rule.
+    text: >
+        az cdn endpoint rule remove -g group -n endpoint --profile-name profile --rule-name Global\\
+"""
+
+helps['cdn endpoint rule show'] = """
+type: command
+short-summary: show delivery rules asscociate with the endpoint.
```

Comment:
show command should only show the target command, right?

Reply:
This command displays all rules engine config associated with the endpoint. We do not have a api call to display one specific rule.